### ISSUE TITLE
Add config option to add alternative python packages

### ISSUE THREAD

**Issue by mike0sv** [iterative/mlem#362]
Sometimes python package name is not the same as it's module name (eg `sklearn` vs `scikit-learn`). For that we have `MODULE_PACKAGE_MAPPING` in `mlem.core.requirements` that you can update from code. But it would be nice to be able to do it from config like
`config.yaml`
```
core:
  package_mapping:
    box: python-box
```

**Comment by alex000kim** [comment]
Would this approach require the user to manually specify the mapping?

**Comment by mike0sv** [comment]
Yes, there is no way to know package name after it was installed

**Comment by aguschin** [comment]
@mike0sv, have you seen [this approach](https://importlib-metadata.readthedocs.io/en/stable/using.html#package-distributions)? I've tried it and it seems to work
```python
>>> from importlib_metadata import packages_distributions
>>> packages_distributions()['box']
['python-box']
```

**Comment by aguschin** [comment]
Got to this again, while trying to use `pip install chatgptpy --upgrade`, which is imported like `from pychatgpt import Chat`. We need to fix this 🔧
